src/api/firebase.js

```diff
@@ -63,12 +59,19 @@ class FirebaseApi {
   }
 
   static databaseSet(path, value) {
+    return firebase.database().ref(path).set(value);
+  }
 
+  static unsubDatabase(path) {
+    return firebase.database().ref(path).off();
+  }
+
+  static databasePathValueLimitToLast(path, limit, handler) {
     return firebase
       .database()
       .ref(path)
-      .set(value);
-
+      .limitToLast(limit)
+      .on('value', handler);
```

Comment:
When working with list of datas, you coud (and should at scale) be a more granular with data loading.

You can check the firebase doc to know how: https://firebase.google.com/docs/database/web/lists-of-data